src/mappings/common.ts

```diff
@@ -99,13 +99,15 @@ export async function cachedCurrentEra(block: SubstrateBlock): Promise<EraIndex>
     }
 }
 
+// Due to memory consumption optimization `eraStakersByEra` contains only one key
 let eraStakersByEra: {[era: number]: [StorageKey<[EraIndex, AccountId]>, Exposure][]} = {}
 
 export async function cachedEraStakers(era: number): Promise<[StorageKey<[EraIndex, AccountId]>, Exposure][]> {
     let cachedValue = eraStakersByEra[era]
     if (cachedValue !== undefined) {
         return cachedValue
     } else {
+        eraStakersByEra = {}
```

Comment:
I think we should try to use database (and benefit it's own cache) because we store EraValidatorInfo entity